app/components/HowCanIHelp/HelpItem.tsx

```diff
@@ -5,16 +5,21 @@ type HelpItemProps = {
   title?: ReactNode
   children: ReactNode
   links?: Link[]
+  titleFont?: 'large-bold' | 'default-bold'
 }
-const HelpItem = ({title, children, links}: HelpItemProps) => {
+const HelpItem = ({title, children, links, titleFont = 'large-bold'}: HelpItemProps) => {
   return (
     <div className="section-split padding-bottom-40">
       <div>
-        {title && <p className="default-bold padding-bottom-16">{title}</p>}
+        {title && <p className={`padding-bottom-16 ${titleFont}`}>{title}</p>}
         <p className="grey default padding-bottom-16"> {children} </p>
       </div>
       <div>
-        {links?.map((link) => <LinkCard key={link.title} action={link.action || '#'} {...link} />)}
+        {links?.map((link) => (
+          <div key={link.title} className="padding-bottom-16">
+            <LinkCard action={link.action || '#'} {...link} />
```

Comment:
why not apply the class directly to the `LinkCard`?

Reply:
Because we are spacing with padding, and the padding would go inside the button